llvm/CMakeLists.txt

```diff
@@ -626,43 +626,43 @@ endif()
 # components which may not strictly be necessary (tools, examples, and tests).
 #
 # This is primarily to support building smaller or faster project files.
-option(LLVM_INCLUDE_TOOLS "Generate build targets for the LLVM tools." ON)
+option(LLVM_INCLUDE_TOOLS "Generate build targets for the LLVM tools." OFF)
```

Comment:
All options are force disabled in "ClickHouse/contrib/llvm-project-cmake/CMakeLists.txt" already. Why disable them again here?

Reply:
During cross compilation LLVM tries to build native table gen. To build it, CMake starts new native project without all options that we specified in CMake, and tries to build table gen.
It is necessary to build native project without options that we specified in CMake, because for main project during cross compilation we specified different architecture.